packages/connect/src/core/AbstractMethod.ts

```diff
@@ -255,7 +255,9 @@ export abstract class AbstractMethod<Name extends CallMethodPayload['method'], P
             return;
         }
         const { device } = this;
-        if (!device.features) return;
+
+        // do not do rfw ange check for devices in BL mode as we do not know fw version of some device models
```

Comment:
what language is this?

Reply:
please specify reason (descripbe model One behavior)

Reply:
updated